src/AstRunner/AstParser/NikicPhpParser/FileReferenceVisitor.php

```diff
@@ -293,7 +299,10 @@ private function enterFunction(string $name, Node\Stmt\Function_ $node): void
     private function getClassReferenceName(ClassLike $node): ?string
     {
         if (isset($node->namespacedName)) {
-            return $node->namespacedName->toCodeString();
+            /** @psalm-var \PhpParser\Node\Name $namespacedName */
```

Comment:
How about moving this to the function doc block and keeping the early return?

Reply:
Not really possible since it is a library function. Would have to write a stub.